daumie/bucketlist-django
# Bucketlist API using Django

### What does this PR do?
This PR brings forth the implementation of an online bucket-list API using the Django Rest Framework.

### How to test this PR?
Install [virtualenv](https://virtualenv.pypa.io/en/stable/) and create a separate python environment by [activating the virtualenv.](http://docs.python-guide.org/en/latest/dev/virtualenvs/)
The project's requirements can be installed using by
```pip install -r requirements.tx```
Run the application by invoking 
```python manage.py runserver```
The expected output is per the images displayed on the [README](https://github.com/daumie/bucketlist-django/blob/develop/README.md#bucketlist-api-and-web_application)
